packages/build/src/packaging/package/redhat.ts

```diff
@@ -38,7 +61,7 @@ export async function createRedhatPackage(
     ...pkg.otherDocFilePaths.map(({ packagedFilePath }) => `%doc ${packagedFilePath}`),
   ];
   if (pkg.manpage) {
-    filelistRpm.push(`%doc ${pkg.manpage.packagedFilePath}`);
+    filelistRpm.push(`%{_mandir}/man${getManSection(pkg.manpage.packagedFilePath)}/${pkg.manpage.packagedFilePath}`);
```

Comment:
Man pages don’t really go into the standard documentation directory – this would have put them into `/usr/share/doc/mongosh/`, which in rpm is called `%{_docdir}`. However, in order for `man` to pick this up, we need to put the file into `%{_mandir}`.

Since that’s similar to the `%{_bindir}`/`%{_libexecdir}` file installment that we do above already, I think it makes sense to add the man page the same way (just with mode 0644 instead of 0755 since it’s not an executable).

The tests failed for me locally because of this – I checked CI, and it seems that we just skip the test there because those machines don’t have `rpmbuild` installed. That makes me a bit nervous, tbh.

In any case, I’ve added some dummy `man` invocations to the docker files that should hopefully catch this in the future.